css/calendarlist.css

```diff
@@ -112,6 +112,9 @@
 #app-navigation .app-navigation-list-item .utils .action span {
 	opacity: .3;
 }
+#app-navigation .app-navigation-list-item .utils .action.withitems span {
+	opacity: .7 !important;
```

Comment:
Yes, just like in the files app :) 

Please excuse my brevity and typos. 
Sent from my mobile

> On Jun 9, 2016, at 1:38 PM, Mathieu Schopfer <notifications@github.com> wrote:
> 
> In css/calendarlist.css:
> 
> > @@ -112,6 +112,9 @@
> >  #app-navigation .app-navigation-list-item .utils .action span {
> >  	opacity: .3;
> >  }
> > +#app-navigation .app-navigation-list-item .utils .action.withitems span {
> > +	opacity: .7 !important;
> The !important rule overrides the :hover selector below, so the icon opacity won't turn to 1 we the mouse hovers it.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>